test/refactoring/MemberwiseInit/class_members.swift

```diff
@@ -0,0 +1,11 @@
+class Person {
+  var firstName: String!
+  var lastName: String!
+  var age: Int!
+  var planet = "Earth", solarSystem = "Milky Way"
+  var avgHeight = 175
+}
+
+// RUN: %empty-directory(%t.result)
+// RUN: %refactor -memberwise-init -source-filename %s -pos=1:8 > %t.result/class_members.swift
+// RUN: diff -u %S/Outputs/class_members/class_members.swift.expected %t.result/class_members.swift
```

Comment:
Could you also add an applicable test in `test/refactoring/RefactoringKind`?  This refactoring shouldn't be available in value reference cases, like when cursor points to Person in `_ = Person()`.

Reply:
@nkcsgexi should this test be written in `basic.swift` file in `/RefactoringKind/` or should I create a new file?

Reply:
I think a new test file will be better.

Reply:
@nkcsgexi So I have a file 
```
class Person {
  var firstName: String!
  var lastName: String!
  var age: Int!
  var planet = "Earth", solarSystem = "Milky Way"
  var avgHeight = 175
}

let _ = Person()

// RUN: %refactor -memberwise-init -source-filename %s -pos=9:10 | %FileCheck %s -check-prefix=CHECK-NONE

// CHECK-NONE: Action begins
// CHECK-NONE-NEXT: Action ends
```

and this seems to be causing an assertion failure: 
`Assertion failed: (RealOffset+Size <= Buffer.size() && "Invalid location"), function RemoveText, file /Users/Labs/Documents/swift-source/llvm/tools/clang/lib/Rewrite/Rewriter.cpp, line 55.`

Any idea what's going on? Am I missing some LLVM command or something?

Reply:
@nkcsgexi the above assertion seems to be failing for the the `extract.swift` file in `/RefactoringKind/` as well...